### PR TITLE
Rename ExitValidatorSetTx to DisableValidatorTx

### PR DESCRIPTION
**Pull request by geoff-vball**
## Why this should be merged
Aligns with most recent changes in ACP-77 https://github.com/avalanche-foundation/ACPs/pull/144

### CODE REVIEW

File: contracts/staking/README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The `SetSubnetValidatorWeightMessage` is delivered to the P-Chain as the payload
...
+ACP-77 also provides a method to disable a validator without interacting with the Subnet directly. The P-Chain transaction [`DisableValidatorTx`](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets#disablevalidatortx) disables the validator on the P-Chain. The disabled validator's weight will still count towards the Subnet's total weight. Disabled Subnet Validators can re-activate at any time by increasing their balance with an `IncreaseBalanceTx`.
```
Comment (minghinmatthewlam) [inline contracts/staking/README.md:39]: can you also clarify taht `IncreaseBalanceTx` can be submitted by any one, not just validator themselves

File: contracts/staking/README.md [hunk 2]
```diff
@@ -34,9 +34,11 @@ Validator exit is initiated with a call to `initializeEndValidation` on the Vali
...
+ACP-77 also provides a method to disable a validator without interacting with the Subnet directly. The P-Chain transaction [`DisableValidatorTx`](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets#disablevalidatortx) disables the validator on the P-Chain. The disabled validator's weight will still count towards the Subnet's total weight. 
+
+Disabled Subnet Validators can re-activate at any time by increasing their balance with an `IncreaseBalanceTx`. Anyone can call `IncreaseBalanceTx` for any validator on the P-Chain. A disabled validator can only be totally removed from the validator set by a call to `initializeEndValidation`.
```
Comment (michaelkaplan13) [inline contracts/staking/README.md:41]: This actually isn't quite correct. 

I a validator's balance runs out on the P-Chain, then they are marked as inactive and anyone can increase their balance to top it up. However after a `DisableValidatorTx`, the only change to that validator that is allow is `SetValidatorWeight = 0`, to completely remove it. You cannot reactive a validator after `DisabelValidatorTx` though.

Comment (minghinmatthewlam) [reply contracts/staking/README.md:41]: I think in the update to ACP77 `DisableValidatorTx` is the same as any other inactive valiator and can be re activated with `IncreaseBalaanceTx`. https://github.com/avalanche-foundation/ACPs/pull/144/files#diff-d2befccafcc84a6c9bf56f76def8e04647ed5087fe8480f28de11fea63fb769aR268

Comment (michaelkaplan13) [reply contracts/staking/README.md:41]: Did not realize this! 

Thank you correcting me here

**Review by cam-schultz — APPROVED** [review]

**Review by iansuvak — APPROVED** [review]

**Pull request merged**